Makefile

```diff
@@ -5,117 +5,57 @@
-#
-#     $(BUILDDIR)/
-#     $(BUILDDIR)/%/
-#         This target simply creates the specified directory. It is limited to
-#         the build-dir as a safety measure. Note that this requires you to use
-#         a trailing slash after the directory to not mix it up with regular
-#         files. Lastly, you mostly want this as order-only dependency, since
-#         timestamps on directories do not affect their content.
-#
-#     FORCE
-#         Dummy target to use as dependency to force `.PHONY` behavior on
-#         targets that cannot use `.PHONY`.
-#
+IMG_BAKE_ARGS ?=
+IMG_BUILDER   ?= default
+IMG_TARGET    ?= osbuild-ci-latest
 
 .PHONY: help
 help:
 	@echo "make [TARGETS...]"
 	@echo
-	@echo "This is the maintenance makefile of osbuild/containers. The"
-	@echo "following targets are available:"
+	@echo "This is the maintenance makefile of osbuild/containers."
+	@echo "The following targets are available:"
 	@echo
-	@echo "            help:       Print this usage information."
-	@echo
-	@echo "       img-setup:       Prepare local docker for image builds"
-	@echo "        img-list:       List image targets as JSON array"
-	@echo "        img-tags:       List image tags as JSON array"
-	@echo "        img-bake:       Build images via docker-buildx-bake"
-
-$(BUILDDIR)/:
-	$(BIN_MKDIR) -p "$@"
-
-$(BUILDDIR)/%/:
-	$(BIN_MKDIR) -p "$@"
-
-.PHONY: FORCE
-FORCE:
-
-#
-# Image Builds
-#
-# The following section provides common helpers around `docker buildx`, which
-# we use to build all images. For more control, you should invoke buildx
-# directly.
-#
-
-IMG_BAKE_ARGS		?=
-IMG_BUILDER		?= default
-IMG_TARGET		?= default
-
-.PHONY: img-setup
-img-setup:
+	@echo "    help:               Print this usage information."
+	@echo "    setup-builder:      Prepare local docker for image builds"
+	@echo "    list-targets:       List all targets from bakefile."
+	@echo "    list-tags:          List all groups from bakefile."
+	@echo "    inspect-target:     Show details of target"
+	@echo "    bake:               Build images via docker-buildx-bake"
+	@echo ""
+	@echo "Additional env variables which can be overwritten from command line:"
+	@echo "IMG_TARGET:      Set image to build / inspect (default osbuild-ci-latest)"
+	@echo "IMG_BAKE_ARGS:   Add extra arguments for docker bake"
+	@echo "IMG_BUILDER:     Set custom builder"
+	@echo ""
+	@echo "Examples:"
+	@echo "make setup-builder"
+	@echo "make list-targets"
+	@echo "make inspect-target IMG_TARGET=osbuild-ci-latest"
+	@echo "make bake IMG_TARGET=osbuild-ci-latest"
+	@echo ""
+	@echo "When no extra parameters are specified IMG_TARGET defaults"
+	@echo "to osbuild-ci-latest"
+
+
+.PHONY: list-targets
+list-targets:
+	@$(BIN_DOCKER) buildx bake --print all-images | jq -c '.target | keys'
+
+.PHONY: list-tags
+list-tags:
+	@$(BIN_DOCKER) buildx bake --print all-images | jq -c '[.target[].tags[]]'
+
+.PHONY: inspect-target
+inspect-target:
+	@$(BIN_DOCKER) buildx bake --print $(IMG_TARGET)
```

Comment:
nitpick: I would suggest to keep the formatting consistent for all targets in the file. This means either having the command on single line or keeping them split to multiple lines as it was before. While the multi-line format is not the prettiest, it makes reviewing diffs much easier. So up to you, just please pick one style.

Reply:
Code has been updated to unify everything as multiline split :)